resources/tests/org.eclipse.core.tests.resources/src/org/eclipse/core/tests/resources/regression/Bug_530868.java

```diff
@@ -64,8 +63,6 @@ public void setUp() throws Exception {
 	public void testMillisecondResolution() throws Exception {
 		assumeFalse("not relevant on Mac, as it does not have milliseconds resolution", OS.isMac());
 		try {
-			assertTrue("can only run if native provider can be enabled", LocalFileNativesManager.setUsingNative(true));
-
```

Comment:
For Windows the new Win32Handler supports millisecond resolution.
And according to this answer https://stackoverflow.com/a/24806116/14542697 respectively https://bugs.openjdk.org/browse/JDK-8177809, millisecond resolution should now also be available through java.nio on Linux.